web/src/styles/global.css

```
@@ -3,6 +3,28 @@
    Uses design tokens from tokens.css
    ========================================================================== */
 
+/* ---------- Self-hosted Inter font ----------
+   Served from /public/fonts to avoid any third-party transfer at page load.
+   Files are the latin and latin-ext variable-font subsets — one file covers
+   all four weights we use (400 / 500 / 600 / 700). */
+@font-face {
+  font-family: 'Inter';
```

Stylelint (17.7.0)

[error] Expected no quotes around "Inter" (font-family-name-quotes)

(font-family-name-quotes)